src/main/java/weavers/siltarae/mistake/service/PeedService.java

```diff
@@ -0,0 +1,28 @@
+package weavers.siltarae.mistake.service;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.data.domain.Page;
+import org.springframework.stereotype.Service;
+import weavers.siltarae.mistake.domain.Mistake;
+import weavers.siltarae.mistake.domain.repository.MistakeRepository;
+import weavers.siltarae.mistake.dto.request.PeedRequest;
+import weavers.siltarae.mistake.dto.response.MistakeListResponse;
+
+@Service
+@RequiredArgsConstructor
+@Slf4j
+public class PeedService {
```

Comment:
```suggestion
@Service
@Transactional
@RequiredArgsConstructor
@Slf4j
public class PeedService {
```

여기도 서비스니까 `@Transactional` 추가해주면 더 좋을것같아요~!

Reply:
넵~~ 👍